bundle/manifests/cockroach-operator.clusterserviceversion.yaml

```
@@ -40,27 +40,61 @@ metadata:
               }
             },
             "image": {
-              "name": "cockroachdb/cockroach:v20.1.5",
+              "name": "registry.connect.redhat.com/cockroachdb/cockroach:v20.2.2",
               "pullPolicy": "Always"
             },
             "nodes": 3,
             "tlsEnabled": true
           }
         }
       ]
-    capabilities: Basic Install
+    capabilities: Full Lifecycle
+    categories: Database
+    certified: "true"
+    containerImage: registry.connect.redhat.com/cockroachdb/cockroachdb-operator:v1.1.0-rc.1
+    createdAt: 2020-12-14T15:44:29Z
+    description: CockroachDB Operator
     operators.operatorframework.io/builder: operator-sdk-v1.1.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
-  name: cockroach-operator.v0.0.10
+    repository: https://github.com/cockroachdb/cockroach-operator
+    support: Cockroach Labs
+  name: cockroach-operator.v1.1.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: CrdbCluster
+    - description: CrdbCluster is the CRD for the cockroachDB clusters API
+      displayName: CoachroachDB Operator
+      kind: CrdbCluster
       name: crdbclusters.crdb.cockroachlabs.com
+      resources:
+      - kind: Service
+        name: cockroach-operator
+        version: v1
+      - kind: StatefulSet
+        name: cockroach-operator
+        version: v1
+      specDescriptors:
+      - description: Number of nodes (pods) in the cluster
```

```suggestion
      - description: Number of database nodes - replicas of a stateful set
```

@taroface can we get some English help here?  What makes sense?

---

Does this refer to the number of pods? If so, I'd say:

`Number of pods in the Kubernetes cluster`